sim/dex-species.ts

```diff
@@ -237,6 +237,13 @@ export class Species extends BasicEffect implements Readonly<BasicEffect & Speci
 
 	constructor(data: AnyObject) {
 		super(data);
+		// FIXME: These shouldn't be necessary, Object.assign in the super should serve this purpose but doesn't
```

Comment:
There are two options to fix this described [here](https://github.com/evanw/esbuild/issues/2284#issuecomment-1141399397)

Reply:
I never would have found this. Thanks so much!